src/main/java/com/github/fge/jsonschema/keyword/validator/common/PatternValidator.java

```diff
@@ -31,13 +34,15 @@
 /**
  * Keyword validator for {@code pattern}
  *
- * @see RhinoHelper
  */
 public final class PatternValidator
     extends AbstractKeywordValidator
 {
+    private HashMap<String, Pattern> _patterns;
+    
     public PatternValidator(final JsonNode digest)
     {
+        _patterns = new HashMap<String, Pattern>();
         super("pattern");
```

Comment:
super should be the first statement in the constructor,  otherwise, compilation fail with 

`call to super must be first statement in constructor`